apps/cms/README.md

```diff
@@ -0,0 +1,37 @@
+![Saleor App CMS Hub](https://user-images.githubusercontent.com/249912/71523206-4e45f800-28c8-11ea-84ba-345a9bfc998a.png)
```

Comment:
readme must be updated, maybe @peelar can help. Mainly app name is just `CMS` or `Saleor CMS app`